### PR TITLE
LB-113: Update robots.txt for Scrobble API endpoint

### PR DESCRIPTION
**Pull request by dpmittal**
Changed the robots.txt file to block path to new Lastfm API and Auth pages
Issue Link: https://tickets.metabrainz.org/browse/LB-113

### CODE REVIEW

File: webserver/static/robots.txt [hunk 1]
```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
+Disallow: /*lastfm
```
Comment (pinkeshbadjatiya) [inline webserver/static/robots.txt]: Why is this line needed?

**Comment by dpmittal** [comment]
@pinkeshbadjatiya Made the changes

File: webserver/static/robots.txt [hunk 2]
```diff
@@ -2,3 +2,5 @@ User-agent: *
 Disallow: /current-status
 Disallow: /user/
 Disallow: /1/
+Disallow: /api
```
Comment (Freso) [inline webserver/static/robots.txt]: Is there a reason for using `/api` here and not `/api/`? (I'm a bit rusty on my robots.txt syntax.)

Comment (dpmittal) [inline webserver/static/robots.txt]: There's a file /api.py which needs to be blocked for bots . /api means every file will be blocked with '/api' where '/api/' will block /api/ folder (which is not present in LB) not a file :)

**Comment by pinkeshbadjatiya** [comment]
This looks good to me.

File: webserver/static/robots.txt (hunk 2 above)
Comment (gentlecat) [inline webserver/static/robots.txt]: You are blocking URLs, not files or folders. It is better to have a trailing slash everywhere for consistency sake and to avoid blocking matching prefixes like `/apistuff`.

Comment (dpmittal) [inline webserver/static/robots.txt]: Actually, I didn't know about robots.txt when I googled about it it said /filename /foldername/ so i thought robots.txt worked like that  but now i came to know that it blocks urls... I have changed the file now

**Comment by dpmittal** [comment]
@mayhem @gentlecat I have made the changes :)
